transports/http/endpoints/routes/routes_config.go

```diff
@@ -23,6 +23,15 @@ type ApiEndpoints interface {
 	RegisterApiEndpoints(router *gin.RouterGroup, cfg *config.HTTPConfig)
 }
 
+// PprofEndpointsFunc wrapping type for function to mark it as implementation of PprofEndpoints.
+type PprofEndpointsFunc func(router *gin.RouterGroup)
+
+// PprofEndpoints registrar which will register routes in root context of application.
+type PprofEndpoints interface {
+	// RegisterPprofEndpoints register Pprof endpoints.
+	RegisterPprofEndpoints(router *gin.RouterGroup)
+}
+
```

Comment:
It is not needed, because you don't apply any custom configuration to those types.
Therefore, you should use `RootEndpoints` (actually `RootEndpointsFunction`) in your handler. No need for additional types

Reply:
removed due to as in comment